pageserver/src/tenant/timeline.rs

```diff
@@ -799,10 +799,16 @@ impl Timeline {
             .await
         {
             Ok((partitioning, lsn)) => {
+                // Use a modified RequestContext that disables atime updates, so that generating image files
+                // doesn't spuriously move old delta layers up the LRU queue for eviction
```

Comment:
Could just comment why:
```suggestion
                // avoid updating access stats for input layers so they remain more evictable, hopefully
```

Reply:
Shortened this comment.